php-sync/etc/sysconfig/sfs/config.php.sample

```diff
@@ -0,0 +1,51 @@
+<?php
+# BUG --xattrs https://bugzilla.samba.org/show_bug.cgi?id=10381
+/*global rsync options used for file transfer
+ * consider adding -H (preserve Hardlinks)
+ * if run as root, you may want to preserve group/owner (-go)
+ * remove "q" if you use debug mode, so you can see the real transfer
+ */
+$RSYNC_OPTS = "-ltpDcuhROq --exclude /.sfs.conf --exclude /.sfs.mounted --delete-missing-args --delete-delay --files-from=%b %s %d";
+//data dir, if this is the same for all nodes
+$DATADIR = '/mnt/data/';
+//batch dir, if this is the same for all nodes
+$BATCHDIR = '/mnt/batches/';
+//drop privileges to user if different from 0, needs a restart if changed
+$UID = 0;
```

Comment:
Ok so the problem is that the two services are running with different users. That's fine then. Once again we have to reinvent things at app level because the service manager isn't able to template `User` and `Group`. Understandable :)